src/Sarif.Viewer.VisualStudio/CodeFlowToTreeConverter.cs

```diff
@@ -23,7 +23,7 @@ internal static List<CallTreeNode> Convert(CodeFlow codeFlow, Run run)
 
                 foreach (ThreadFlowLocation location in threadFlow.Locations)
                 {
-                    ArtifactLocation artifactLocation = location.Location.PhysicalLocation?.ArtifactLocation;
+                    ArtifactLocation artifactLocation = location.Location?.PhysicalLocation?.ArtifactLocation;
```

Comment:
>on?.Phy [](start = 71, length = 7)

I assume this was the actual bug fix.

Reply:
I changed this after updating. So, when I started to check the unit test, they were failing. And with that, they started to pass.